project/EnsimeBuild.scala

```diff
@@ -55,7 +55,8 @@ object EnsimeBuild {
       "org.apache.lucene" % "lucene-core" % luceneVersion
     ),
 
-    updateOptions := updateOptions.value.withCachedResolution(true)
+    updateOptions := updateOptions.value.withCachedResolution(true),
+    resolvers += Resolver.sonatypeRepo("snapshots")
```

Comment:
ouch, for scala-refactoring? it would be good to hold off until they have a final release